backoff.py

```diff
@@ -218,10 +218,22 @@ def constant(interval=1):
         yield interval
 
 
+def random_jitter(value):
+    return value+random.random()
```

Comment:
Although this raises the question why the travis check passed. I'll look into that.